demos/olympia_db/polymer/demo.html

```diff
@@ -3,16 +3,15 @@
 <head>
   <meta charset="UTF-8">
   <title>Polymer Lovefield Demo</title>
-
-  <link rel="import" href="../bower_components/core-ajax/core-ajax.html">
+  <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
+  <link rel="import" href="../bower_components/iron-ajax/iron-ajax.html">
+  <link rel="import" href="../bower_components/iron-flex-layout/classes/iron-flex-layout.html">
```

Comment:
"../bower_components" should be "bower_components".

The only bower_components folder is created at the same directory as the demo.html file.